tutorials/time-freq/ssvep.py

```diff
@@ -306,7 +329,7 @@ def snr_spectrum(psds, noise_n_neighborfreqs=1, noise_skip_neighborfreqs=1):
 # - Alternatively, one can subset trials already at the epoch level using MNEs
 #   event information, and create individual PSD and SNR objects.
 #
-# Here we have already subsetted trials before snr calculation (only 12Hz
+# Here we have already sub-setted trials before snr calculation (only 12Hz
```

Comment:
```suggestion
# Here we have already subsetted trials before snr calculation (only 12Hz
```

Quite sure that is incorrect (see [here](https://en.wiktionary.org/wiki/subset)). Don't care enough to press on it if that is your preferred spelling though.